priv/perf/apps/load_test/lib/test_runner.ex

```diff
@@ -0,0 +1,49 @@
+defmodule LoadTest.TestRunner do
+  alias ExPlasma.Encoding
+
+  @tests %{
+    "deposits" => {LoadTest.Runner.Deposits, LoadTest.Scenario.Deposits}
+  }
+
+  @configs %{
```

Comment:
currently, params are hardcoded. should I make them configurable?

Reply:
@ayrat555 it's entirely up to your discretion, this *entire story* is part of work that **you're owning** and are **responsible**. if you make a change - it needs to make sense based on the requirements **you** provided and we **all did at the beginning**. 

I don't know what this `configs` does, so you'll need to clarify if you're unsure.

Reply:
these values are used in test:
token - token address used for deposits
initial_amount - the amount that the depositor gets from the faucet
deposited_amount - the amount that the depositor deposits to the childchain
transferred_amount - the amount that the depositor send to another account on the childchain